src/hotspot/share/services/nmtPreInit.hpp

```diff
@@ -30,6 +30,7 @@
 #ifdef ASSERT
 #include "runtime/atomic.hpp"
 #endif
+#include "services/memTracker.hpp" // for MemTracker::is_initialized()
```

Comment:
Is this comment really needed?

Reply:
> Is this comment really needed?

Probably not. I will remove the comment.